Sources/SemanticAnalyzer/SemanticAnalyzer+Expressions.swift

```diff
@@ -61,26 +71,33 @@ extension SemanticAnalyzer {
     return ASTPassResult(element: attemptExpression, diagnostics: diagnostics, passContext: passContext)
   }
 
-  public func process(functionCall: FunctionCall, passContext: ASTPassContext) -> ASTPassResult<FunctionCall> {
-    let environment = passContext.environment!
+  public func process(arrayLiteral: ArrayLiteral, passContext: ASTPassContext) -> ASTPassResult<AST.ArrayLiteral> {
```

Comment:
You don’t need to namespace ArrayLiteral here (drop AST.)

Reply:
This is just git not matching the changes, AST. was already here.